p2p_wallet/Scenes/CreateOrRestoreWallet/CreateWallet/CreateSecurityKeys/CreateSecurityKeys.KeysView.swift

```diff
@@ -0,0 +1,97 @@
+//
+// Created by Giang Long Tran on 04.11.21.
+//
+
+import Foundation
+import BECollectionView
+import RxSwift
+import RxCocoa
+
+extension CreateSecurityKeys {
```

Comment:
@bigearsenal 

Hi, can you check the collection view and give me a feedback?

Reply:
It's ok